BankAccountApp/BankAccountApp/BankAccount.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BankAccountApp
+{
+    class BankAccount : GeneralAccount, IAccountOperations
+    {
+        public enum ACCOUNT_STATUS { OPEN, CLOSED };
+
+        private decimal balance;
+        private Owner owner;
+        private ACCOUNT_STATUS accountStatus;
+        private List<Operation> listaOperatii;
+
+        public List<Operation> ListaOperatii
+        {
+            get { return listaOperatii; }
+            private set { listaOperatii = value; }
+        }
+
+
+        public ACCOUNT_STATUS AccountStatus
+        {
+            get { return accountStatus; }
+            set { accountStatus = value; }
+        }
+
+
+        public Owner Owner
+        {
+            get { return owner; }
+            set { owner = value; }
+        }
+
+        public decimal Balance
```

Comment:
Parca as mai vedea si o operatiune de genul: CheckBalance, care sa returneze cat este in cont acum

Reply:
Implementez. De acord.